src/integration-tests/model-in-image/deploy-traefik.sh

```diff
@@ -49,32 +53,60 @@ set +e
 helm get values ${TRAEFIK_NAME} -n ${TRAEFIK_NAMESPACE} > $WORKDIR/test-out/traefik-values.cur 2>&1
 res=$?
 set -e
-echo ${DOMAIN_NAMESPACE} >> $WORKDIR/test-out/traefik-values.cur
+for evar in DOMAIN_NAMESPACE TRAEFIK_NAMESPACE TRAEFIK_NAME TRAEFIK_HTTP_NODEPORT TRAEFIK_HTTPS_NODEPORT; do
+  echo "${evar}=${!evar}" >> $WORKDIR/test-out/traefik-values.cur
+done
 if [ $res -eq 0 ] \
    && [ -e "$WORKDIR/test-out/traefik-values.orig" ] \
    && [ "$(cat $WORKDIR/test-out/traefik-values.cur)" = "$(cat $WORKDIR/test-out/traefik-values.orig)" ]; then
   echo "@@"
-  echo "@@ Traefik already installed. Skipping uninstall/install."
+  echo "@@ Traefik already installed. Skipping uninstall/install. Current values:"
+  cat $WORKDIR/test-out/traefik-values.orig
   echo "@@"
 else
   set +e
+  echo "@@"
+  echo "@@ Uninstalling old install (if any) using 'helm uninstall $TRAEFIK_NAME -n $TRAEFIK_NAMESPACE'"
+  echo "@@"
   helm uninstall $TRAEFIK_NAME -n $TRAEFIK_NAMESPACE
+  echo "@@ Creating traefik namespace '$TRAEFIK_NAMESPACE' and domain namepace '$DOMAIN_NAMESPACE'"
   kubectl create namespace $TRAEFIK_NAMESPACE
   kubectl create namespace $DOMAIN_NAMESPACE
   set -e
 
   cd ${SRCDIR}
 
+  echo "@@ Installing traefik"
+
   # you only need to add the repo once, but we do it every time for simplicity
   helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 
+  set -x
+
   helm install ${TRAEFIK_NAME} stable/traefik \
```

Comment:
Can you use the the script setup.sh  in samples/charts/utils directory to configure traefik.  The latest version of traefik  ( v 2.2.1) has different chart repository ( containous.github.io/traefik-helm-chart), The chart name is not stable/traefik but traefik/traefik.  I am working on a PR to upgrade the traefik version.  All these will be taken care by setup.sh, provided you use the command "setup.sh create traefik" in your script.

Reply:
How about I do that in a future PR once the newest version of setup.sh is ready?

Reply:
Note: The interface to setup.sh will not change unless you want a non default Traefik version.

Reply:
We will wire in setup.sh in a different pull, as discussed in the meeting.  Please approve if you're fine with this change.